packages/forms-web-app/src/validators/lpa/enter-code.js

```diff
@@ -0,0 +1,27 @@
+const { body } = require('express-validator');
+
+const ruleEnterCode = () =>
+	body('email-code')
+		.notEmpty()
+		.withMessage('Enter the code')
+		.bail()
+		.isLength({ min: 5 })
+		.withMessage('You’ve not entered enough characters, the code must be 5 characters')
+		.bail()
+		.isLength({ max: 5 })
+		.withMessage('You’ve entered too many characters, the code must be 5 characters')
+		.bail()
+		.custom((value) => {
+			return !(value.match(/[^\w_]/g) || value.match(/[AEIOUaeiou0]/g));
```

Comment:
Should we put this token regex into a constant somewhere that can be accessed by all packages that need it so we only need to do it one place or is that overkill at this point considering we only have it in 2 files?

Reply:
It certainly couldn't hurt and is better practice.